app/views/layout.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Chitter</title>
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
+    <link href="/normalize.css" rel="stylesheet">
+    <link href="/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="everything">
+      <header>
+      <% if current_user %>
```

Comment:
I would probably make this two different views (and return a different view from the controller depending omn the sign in state) rather than an if/else in the view. It makes it explicit earlier in the flow that there is a signed in/signed out state (which helps with separation of logical concerns) and would also make the views easier to read.